admin/cmd/bootstrap.go

```diff
@@ -187,6 +190,10 @@ func finishBootstrap(globalFlags *flags.GlobalFlags) {
 
 	agentlocal.SetTransport(ctx, globalFlags.EnableDebug || globalFlags.EnableTrace, globalFlags.PMMAgentListenPort)
 
+	if !globalFlags.DisableVersionCheck {
```

Comment:
We need to make sure this runs only for the `pmm-admin` command, not `pmm` or `pmm-server-upgrade`